components/document-footer-metadata/README.md

```diff
@@ -7,8 +7,75 @@ DocumentFooterMetadata
 ```
 <!-- STORY -->
 
+### Usage
 
 
+Simple
+```js
+import { asAnchor } from '@govuk-react/hoc';
+const AnchorTag = asAnchor('a');
+
+const fromData = [
+  <AnchorTag href="/government/organisations/ministry-of-defence">
+    Ministry of Defence
+  </AnchorTag>,
+];
+
+
+<DocumentFooterMetadata from={fromData} />
+```
+
+
+DFM From & part of example
+```js
+import { asAnchor } from '@govuk-react/hoc';
```

Comment:
I think a blank line between the import and the usage is nice, it's what I did on the navigation components, what do you think we should normalise to?